css/style.css

```diff
@@ -46,15 +46,15 @@ caption {
 }
 
 red {
-    color: #ff0000
+    color: red;
 }
 
 green {
-    color: #00ff00
+    color: lime;
```

Comment:
`green` defaults to `rgb (0, 128, 0)` and not `rgb (0, 255, 0)`.  Therefore `lime` has to be used as an abstract term in css.